docs/usage/2-manifest.mdx

```diff
@@ -98,47 +99,238 @@ default: "[]"
+},
+{
+name: "Current version",
+description: "Current version without the extension.",
+type: "String",
+optional: false,
+default: ""
+}
+]}/>
+
+
+### Platform
+
+The platform type represents the platform a target is built for. It can be any of the following types:
+
+<EnumTable cases={[
+{
+case: ".app",
+description: "An application."
+},
+{
+case: ".staticLibrary",
+description: "A static library."
+},
+{
+case: ".dynamicLibrary",
+description: "A dynamic library."
+},
+{
+case: ".framework",
+description: "A dynamic framework."
+},
+{
+case: ".staticFramework",
+description: "A static framework. This is a regular framework product however is configured to be statically linked."
+},
+{
+case: ".unitTests",
+description: "A unit tests bundle."
+},
+{
+case: ".uiTests",
+description: "A UI tests bundle."
+},
+]}/>
+
+### Target Action
+
+Target actions, represented as target script build phases, are useful to define actions to be executed before of after the build process of a target. 
+
+<EnumTable cases={[
+{
+case: ".pre(tool: String, arguments: String..., name: String)",
+description: "Action executed before the target-specific build phases where tool is the name of the tool to be executed."
+},
+{
+case: ".pre(name: String, arguments: String..., name: String)",
+description: "Action executed before the target-specific build phases where path is the path to the tool to be executed."
+},
+{
+case: ".post(tool: String, arguments: String..., name: String)",
+description: "Action executed after all the target-specific build phases where tool is the name of the tool to be executed."
+},
+{
+case: ".post(name: String, arguments: String..., name: String)",
+description: "Action executed after all the target-specific build phases where path is the path to the tool to be executed."
+},
+]}/>
+
+The following example shows the definition of an action that runs the `my_custom_script.sh` passing the argument `"hello"`:
 
 ```swift
 .pre(path: "my_custom_script.sh", arguments: ["hello"], name: "My Custom Script Phase")
 ```
 
-> Note: Either the tool or the path need to be defined. Moreover, Tuist verifies whether the launch path is valid and fail otherwise.
 
-- **Dependencies (optional):** You can read more about dependencies [here](./dependencies.md)
-- **Settings (optional):** Read more about [settings](#settings)
-- **CoreDataModels (optional):** An array of `CoreDataModel` objects. A `CoreDataModel` is an special type of resource that requires the following two attributes:
-  - **Path:** Relative path to the Core Data model.
-  - **CurrentVersion:** Current version without the extension.
-- **Environment (optional):** It's a `[String: String]` to defined variables that will be set to the scheme that Tuist automatically generates for the target.
+<Message info title="Relative paths" description="Either the tool or the path need to be defined. Moreover, Tuist verifies whether the launch path is valid and fail otherwise."/>
 
 ### Scheme
```

Comment:
Is the idea to incrementally adopt the table for the various sections? e.g. Scheme still uses the list style

Reply:
I missed this one. It should be ported now to the new format 👍